.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior. Please include any code snippet or a link to your project repository that experience the issue so we could better understand the issue.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Actual behavior**
+A clear and concise description of what actually happened.
+
+**Node env (please complete the following information):**
+ - fullstory-babel-plugin-annotate-react [Package version](https://github.com/fullstorydev/fullstory-babel-plugin-annotate-react/tags) [e.g. `v.2.3.0`]
+ - Dev OS: [e.g. MacOS]
+ - Node version (`node -v`) [e.g. `v18.4.0`]
+ - NPM version (`npm -v`) [e.g. `8.12.1`]
+
+**Additional comment**
+Add any other comment about the issue, or the reason and use case for this.
```

Comment:
```suggestion
Add any other comments about the issue.
```

Reply:
fixed 👍

Reply:
can you see it now? Idk if github was bugging out there. 🤔